pkg/remediator/watch/manager.go

```diff
@@ -327,3 +341,65 @@ func (m *Manager) stopWatcher(gvk schema.GroupVersionKind) {
 	w.Stop()
 	delete(m.watcherMap, gvk)
 }
+
+func (m *Manager) startWatchingCRD(gvk schema.GroupVersionKind, commit string) {
+	m.crdController.SetReconciler(gvk.GroupKind(), func(ctx context.Context, crd *apiextensionsv1.CustomResourceDefinition) error {
+		if customresource.IsEstablished(crd) {
+			if err := discoverResourceForKind(m.mapper, gvk); err != nil {
+				// Trigger retry by controller-manager
```

Comment:
We should log the error

Reply:
The error is returned to the CRDController which returns it from the Reconcile call, which triggers the ControllerManager to log it and schedule a retry (with backoff I think). So we don't need to log it here.